### ISSUE TITLE
Serialization for models with custom layer/loss/activation

### ISSUE THREAD

**Issue by gokceneraslan** [Avsecz/keras-hyperopt#5]
As described [here](https://keras.io/getting-started/faq/#how-can-i-save-a-keras-model), `custom_objects` argument of `keras.models.load_model` must be used for loading models with custom loss/layer/activation function. Adding here as a TODO item.

**Comment by Avsecz** [comment]
- [x] add `custom_objects` to 
  - https://github.com/Avsecz/keras-hyperopt/blob/8a2060a5bebb4826ca1da46c622f7d5cbf8a7085/kopt/hyopt.py#L31 and 
  - https://github.com/Avsecz/keras-hyperopt/blob/8a2060a5bebb4826ca1da46c622f7d5cbf8a7085/kopt/hyopt.py#L77
  - https://github.com/Avsecz/keras-hyperopt/blob/8a2060a5bebb4826ca1da46c622f7d5cbf8a7085/kopt/hyopt.py#L262
  - https://github.com/Avsecz/keras-hyperopt/blob/8a2060a5bebb4826ca1da46c622f7d5cbf8a7085/kopt/hyopt.py#L368
 - ...

**Comment by Avsecz** [comment]
Maybe useful for you as well: I typically load all my own custom_objects to 'global custom objects' for convenience (you don't need to specify custom_objects in load_model then):
https://github.com/gagneurlab/concise/blob/master/concise/__init__.py#L22-L27

**Comment by Avsecz** [comment]
I'm thinking whether it's maybe better for the user to just specify custom_objects to the global environment

```
from keras.utils.generic_utils import get_custom_objects
get_custom_objects()["myobject"] = myobject
```

I'll anyway add custom_objects to `CompileFN`, `test_fn`,

**Comment by gokceneraslan** [comment]
> Maybe useful for you as well: I typically load all my own custom_objects to 'global custom objects' for convenience (you don't need to specify custom_objects in load_model then):
> https://github.com/gagneurlab/concise/blob/master/concise/__init__.py#L22-L27

I didn't know about that, very cool!

> I'll anyway add custom_objects to CompileFN, test_fn,

That's also convenient, I think. Thanks!

**Comment by Avsecz** [comment]
closed by https://github.com/Avsecz/keras-hyperopt/pull/6
